services/azure/Compute/virtualMachines.yml

```diff
@@ -40,45 +47,79 @@ privileges:
     notes: >-
       Stops and releases allocation; VMs become unavailable until started.
   runCommand:
-    risks: [exfiltration:data, impact:hijack]
+    risks:
+      [
+        exfiltration:data,
+        exfiltration:crypto,
+        impact:hijack,
+        persistence:account,
+      ]
     notes: >-
       Executes arbitrary commands inside the guest OS, enabling data theft or tampering.
   login:
-    risks: [exfiltration:data, impact:hijack]
+    risks:
+      - collection:data
+      - destruction:data
+      - discovery:network
+      - discovery:policy
+      - escalation:lateral
+      - exfiltration:data
+      - impact:defacement
+      - impact:hijack
```

Comment:
It seems to me like logging in is exactly what they're talking about under `Initial Access > Valid Accounts > Cloud Accounts`.  

These others seem downstream from that? Like if you can login and the machine is wide open then every risk in the catalog could apply here.

>https://attack.mitre.org/techniques/T1078/004/

Reply:
I would at least add a nuanced explanation here like the GCP oslogin one has:

>Specific risks depend on the instance configuration...